north/src/runtime/process.rs

```diff
@@ -501,32 +486,29 @@ pub mod minijail {
         container: &Container,
         run_dir: &Path,
     ) -> Result<()> {
-        if let Some(ref resources) = container.manifest.resources {
-            for res in resources {
-                let shared_resource_path = {
-                    let dir_in_container_path: PathBuf = res.dir.clone().into();
-                    let first_part_of_path = run_dir.join(&res.name).join(&res.version.to_string());
-
-                    let src_dir = dir_in_container_path
-                        .strip_prefix("/")
-                        .map(|dir_in_resource_container| {
-                            first_part_of_path.join(dir_in_resource_container)
-                        })
-                        .unwrap_or(first_part_of_path);
-
-                    if src_dir.exists().await {
-                        Ok(src_dir)
-                    } else {
-                        Err(anyhow!(format!(
-                            "Resource folder {} is missing",
-                            src_dir.display()
-                        )))
-                    }
-                }?;
+        for (target, res) in container.manifest.resources() {
```

Comment:
The `setup_resources` should be merged in to `setup_mounts` which matches on the mount type:
* `Bind`
* `Persist`
* `Resource`